README.md

```diff
@@ -49,6 +49,11 @@ pure-Rust implementations (see `Cargo.toml` for a quick overview).
 
 ### Providers
 
+#### Default
+The default resolver, as mentioned above, uses pure Rust implementations of its functions. It currently has full support for either
+AES-GCM encryption or ChaChaPoly. When using AES-GCM, it is possible to get noticable speedups on modern CPUs due to hardware-backed
+operations. For more information and to see how to use these, see RustCrypto's [aes_gcm](https://docs.rs/aes-gcm/0.1.0/aes_gcm/#performance-notes) docs.
```

Comment:
I don't think this addition is necessary (I'd like to keep READMEs as succinct as possible), and instead perhaps we can just rename the `### Providers` section to `### Other providers`. We may also, eventually, want to go down the rabbit-hole of providing benchmarks rather than mentioning details about the inner implementations hardware acceleration support.

Reply:
That sounds good to me. For now I removed the section and just titled it "Other providers".